src/blockchains/eth/lib/end_of_block.ts

```diff
@@ -1,22 +1,10 @@
-import { ETHBlock } from "../eth_types";
+import { ETHBlock, ETHTransfer } from "../eth_types";
 import { Web3Interface, safeCastToNumber } from "./web3_wrapper";
 
-export type EOB = {
```

Comment:
Do we need the special EOB type? It seems we want it to match the ETHTransfers type structure exactly, so maybe it should not be a special type?